Description: Do not check if we are in a sage source directory
 This is useful for example to run the test suite with the installed sage.
Author: Tobias Hansen <thansen@debian.org>
Forwarded: not-needed

--- a/sage/src/bin/sage
+++ b/sage/src/bin/sage
@@ -360,18 +360,6 @@
 
 # Prepare for running Sage, either interactively or non-interactively.
 sage_setup() {
-    # Check that we're not in a source tarball which hasn't been built yet (#13561).
-    if [ ! -d "$SAGE_LOCAL/lib/python2.7/dist-packages/sage" ]; then
-        echo >&2 '************************************************************************'
-        echo >&2 'It seems that you are attempting to run Sage from an unpacked source'
-        echo >&2 'tarball, but you have not compiled it yet (or maybe the build has not'
-        echo >&2 'finished). You should run `make` in the Sage root directory first.'
-        echo >&2 'If you did not intend to build Sage from source, you should download'
-        echo >&2 'a binary tarball instead. Read README.txt for more information.'
-        echo >&2 '************************************************************************'
-        exit 1
-    fi
-
     # Display the startup banner (unless SAGE_BANNER is explictly "no")
     sage_banner
 
